### DISCUSSION TITLE
自动部署命令在哪里修改

### DISCUSSION THREAD

**Discussion by CarrotVegeta** [Qexo/Qexo#366] [Q&A] [answered]
请问一下自动部署命令在哪修改

**Comment by Fgaoxing** [comment, marked as answer]
不能按时间排序



在 2023-04-26 00:16:46，"胡萝卜贝吉塔" ***@***.***> 写道：

请问一下自动部署命令在哪修改 还有文章/文章列表 怎么按照时间排序

**Comment by am-abudu** [comment]
在Github Action的Workflow File中修改

**Reply by CarrotVegeta** [reply]
![image](https://user-images.githubusercontent.com/34666827/236636645-a95168b6-45a1-4ec7-aae2-d4cf59bf5d80.png)
这个  只有自己去自己github的workflow file文件修改吗  我以为程序自带有这个命令
